llvm/unittests/Transforms/Instrumentation/MemProfUseTest.cpp

```diff
@@ -101,4 +101,116 @@ declare !dbg !19 void @_Z2f3v()
   EXPECT_THAT(CallSites[2],
               Pair(FieldsAre(2U, 9U), IndexedMemProfRecord::getGUID("_Z2f3v")));
 }
+
+TEST(MemProf, ExtractDirectCallsFromIRInline) {
+  // The following IR is generated from:
+  //
+  // void f1();
+  // static inline void f2() { f1(); }
+  // static inline void f3() { f2(); }
```

Comment:
If we keep the function on one line then the offset is always zero. Then it's not possible to test if it was unset or actually zero. Can you put some of these on different lines so that we get non-zero offsets?

Reply:
I've made line/column numbers more interesting in the latest revision.